src/cluster/replication.cc

```diff
@@ -236,18 +237,23 @@ void ReplicationThread::CallbacksStateMachine::Start() {
     handlers_.emplace_front(CallbacksStateMachine::WRITE, "auth write", authWriteCB);
   }
 
+  uint64_t last_connect_timestamp = 0, connect_timeout_ms = 3100;
```

Comment:
If `connect_timeout_ms` is not dynamically updated, it would be better to use constants?

Reply:
I had thought about using a constant before, but found that it wasn't used elsewhere, so I wrote it as a local variable to make it more intuitive.